portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,88 @@
+                technology, when I was in high school I was passionate about coding to the extent that I once saved up
+                some money to buy a small phone which I could use to
+                learn how to code online. The amazing thing is now I get to do this daily!
+            </p>
+        </div>
+        <div id="projects-container">
+            <h2 style="color: blue;">Projects I have worked on:</h2>
+            <ul style="list-style-type: circle;">
+                <li>ZiFarm- a website that connects farmers with customers</li>
+                <li>Drawing App- <a style="text-decoration: none;" href="https://tinotenda6.github.io/drawing/Drawing%20App/">Check it out here</a></li>
+                <li>Math Game- <a style="text-decoration: none;" href="https://tinotenda6.github.io/mathgme/">Check it out here!</a></li>
+            </ul>
+        </div>
+        <div id="Fav-pictures">
+            <h2 style="color: blue;">Check out my favourite pictures I have taken in the period of 2020-2021</h2>
+            <div class="slideshow-container">
+                <div class="mySlides fade">
+                    <div class="numbertext">1 / 5</div>
+                    <img src="images/random_photography/GC_sunrise.jpg" style="width:100%">
+                    <div class="text">Beautiful sunrise in Grinnell</div>
+                </div>
+
+                <div class="mySlides fade">
+                    <div class="numbertext">2 / 5</div>
+                    <img src="images/random_photography/GC_squirrell.jpg" style="width:100%">
+                    <div class="text">Personal favorite</div>
+                </div>
+
+                <div class="mySlides fade">
+                    <div class="numbertext">3 / 5</div>
+                    <img src="images/random_photography/GC_Summer.jpg" style="width:100%">
+                    <div class="text">I totally love this one!</div>
+                </div>
+                <div class="mySlides fade">
+                    <div class="numbertext">4 / 5</div>
+                    <img src="images/random_photography/GC_Winter_squirell.jpg" style="width:100%">
+                    <div class="text">Hope the squirell wasn't feeling too cold!</div>
+                </div>
+                <div class="mySlides fade">
+                    <div class="numbertext">5 / 5</div>
+                    <img src="images/random_photography/Tee_quarantine.jpg" style="width:100%;" >
+                    <div class="text">#Mask_up</div>
+                </div>
+            </div>
+            <br>
+
+           <div style="text-align:center">
+                <span class="dot"></span> 
+                <span class="dot"></span> 
+                <span class="dot"></span> 
+                <span class="dot"></span> 
+                <span class="dot"></span> 
+            </div>
+        
+        </div>
+    
+    
     </div>
-  </body>
-</html>
+</body>
+<script>var slideIndex = 0;
```

Comment:
I'm not sure what the best practice is for js, please feel free to ignore this comment 😅

Would it still work if we move this into the `script.js`? It's a little hard to keep track of the variables across files.

Reply:
Yes it would work, thanks!